lib/utils.js

```diff
@@ -129,6 +139,33 @@ module.exports = {
         throw new Error(err);
       }
     }),
+  
+  /**
+   * Recursively delete a directory 
+   * @param {string} dir 
+   */
+  removeDirectory: function removeDirectory(dir) {
```

Comment:
you don't need to redeclare the function name here, method shorthand in an object will correctly apply a name to it

```suggestion
  removeDirectory: (dir) => {
```

Reply:
![image](https://user-images.githubusercontent.com/6213695/78220975-cc66fb80-74c2-11ea-8c9d-4254672ad08e.png)

this was what I was using as a quick drop in to remove a dir async without the hassle.
Once we go to node@12, we can just delete this drop in